trade-finance-manager-api/src/v1/controllers/deal.tasks.api-test.js

```diff
@@ -9,9 +9,11 @@ const CONSTANTS = require('../../constants');
 const MOCK_DEAL_MIA = require('../__mocks__/mock-deal-MIA-submitted');
 const { createTasks } = require('../helpers/create-tasks');
 const mapSubmittedDeal = require('../mappings/map-submitted-deal');
+const { MOCK_PORTAL_USERS } = require('../__mocks__/mock-portal-users');
```

Comment:
I know this is legacy, but are the tests in this folder api tests or unit tests? if its not api, could we update the name

Reply:
on looking at this, it appears to be some not very nice middle ground...happy to leave

Reply:
I *think* it is a unit test